scripts/index.js

```diff
@@ -0,0 +1,80 @@
+const cityName = document.querySelector('.city-name');
+const cityNameForm = document.querySelector('#city-name-form');
+const temp = document.querySelector('#temp');
+const plus = document.querySelector('#plus');
+const minus = document.querySelector('#minus');
+const landscape = document.querySelector('.landscape');
+const skyBackground = document.querySelector('.sky-display');
+const skyMenu = document.querySelector('.sky-menu');
+const skyMenuForm = document.querySelector('#sky-menu-form');
+
+cityNameForm.addEventListener('submit', (event) => {
+    let info = new FormData(cityNameForm)
+    for (var value of info.values()) {
+        cityName.textContent = value;
+    } 
+    cityNameForm.style.display = 'none';  
+    event.preventDefault();
+});
+
+cityName.addEventListener('click', (event) => {
+    cityNameForm.style.display = 'block';  
+
+});
+
+const emojiDisp = function (temp_num){
+    switch(true){
+        case (temp_num <= 40):
+            temp.style.color = 'teal'; 
+            break;
+        case (temp_num <= 59):
+            temp.style.color = 'blue'; 
+            landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+            break;
+        case (temp_num <= 69):
+            temp.style.color = 'goldenrod'; 
+            landscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+            break;
+        case (temp_num <= 79):
+            temp.style.color = 'orange'; 
+            landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+            break;
+        case (temp_num > 79):
+            temp.style.color = 'red'; 
+            landscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    }
+}
+
+plus.addEventListener('click', (event) => {
+    let temperature = parseInt(temp.textContent)
+    temp.textContent = temperature + 1
+    emojiDisp(temperature + 1)
+    });
+
+minus.addEventListener('click', (event) => {
+    let temperature = parseInt(temp.textContent)
+    temp.textContent = temperature - 1
+    emojiDisp(temperature - 1)
+    });
+
+skyMenuForm.addEventListener('submit', (event) => {
```

Comment:
This works! If we look at the README, though, we want to select an option and immediately see a change, not wait for a submission button. We could look into how to grab the selected value of a dropdown menu using vanilla javascript